test/pinpoints/instructions/bear_left_guide_sign.txt

```diff
@@ -1 +1 @@
--j '{"locations":[{"lat":38.870739,"lon":-77.154732,"street":"Leesburg Pike"},{"lat":38.871666,"lon":-77.163300,"street":"US 50"}],"costing":"auto","units":"miles"}'
+-j '{"locations":[{"lat":38.870739,"lon":-77.15465,"street":"Leesburg Pike"},{"lat":38.871666,"lon":-77.163300,"street":"US 50"}],"costing":"auto","units":"miles"}'
```

Comment:
when i ran this on new data it was in the middle of a double digitized section of road and snapped to the wrong side so i move it to snap to the right spot less ambiguously